include/swift/Syntax/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+set(SWIFT_GYB_FLAGS
+  --line-directive "''")
```

Comment:
I can't understand why you'd disable line directives like this; the result would be that any compilation errors point into the generated file instead of the original one, with the typical result that maintainers edit the wrong file.

Reply:
Ahhh yes sorry about that. I turned line directives off while initially writing the gyb templates because it made it harder to see what the template instantiation was doing. That said, they should have been re-enabled before merging. I’ll re-enable them in a patch.

Reply:
Now that I'm looking, these are for the C++ bindings. `gyb`'s line directive code emits Swift-style line directives -- clang will just ignore them. The patch for this would need to teach `gyb` how and when to emit C-style line directives, which wouldn't be too difficult.

Reply:
Actually, those aren't swift-style line directives, which are broken for this purpose because they are only valid in certain syntactic positions.  These are just comments that get post-processed by the `line-directive` tool.  For error messages, it doesn't matter that clang would ignore them because the output of clang would get passed through the `line-directive` tool just like the output of Swift does.  However, generating real `#line` directives should also allow you to debug in the `.gyb` source rather than in the generated `.cpp`